docs/source/cpp/compute.rst

```diff
@@ -0,0 +1,419 @@
+
+* \(1) Output is the physical length in bytes of each input element.
+
+* \(2) Each ASCII character in the input is converted to lowercase or
+  uppercase.  Non-ASCII characters are left untouched.
+
+* \(3) Each UTF8-encoded character in the input is converted to lowercase or
+  uppercase.
+
+Containment tests
+~~~~~~~~~~~~~~~~~
+
++--------------------------+------------+----------------------------------+-----------------------+--------------------------------------------+
+| Function name            | Arity      | Input types                      | Output type           | Options class                              |
++==========================+============+==================================+=======================+============================================+
+| binary_contains_exact    | Unary      | String-like                      | Boolean (1)           | :struct:`BinaryContainsExactOptions`       |
++--------------------------+------------+----------------------------------+-----------------------+--------------------------------------------+
+| isin                     | Unary      | Binary- and String-like          | Boolean (2)           | :struct:`SetLookupOptions`                 |
++--------------------------+------------+----------------------------------+-----------------------+--------------------------------------------+
+| isin                     | Unary      | Null                             | Boolean (2)           | :struct:`SetLookupOptions`                 |
++--------------------------+------------+----------------------------------+-----------------------+--------------------------------------------+
+| isin                     | Unary      | Boolean,Numeric, Temporal        | Boolean (2)           | :struct:`SetLookupOptions`                 |
++--------------------------+------------+----------------------------------+-----------------------+--------------------------------------------+
+| match                    | Unary      | Binary- and String-like          | Int32 (3)             | :struct:`SetLookupOptions`                 |
++--------------------------+------------+----------------------------------+-----------------------+--------------------------------------------+
+| match                    | Unary      | Null                             | Int32 (3)             | :struct:`SetLookupOptions`                 |
++--------------------------+------------+----------------------------------+-----------------------+--------------------------------------------+
+| match                    | Unary      | Boolean,Numeric, Temporal        | Int32 (3)             | :struct:`SetLookupOptions`                 |
++--------------------------+------------+----------------------------------+-----------------------+--------------------------------------------+
+
+* \(1) Output is true iff :member:`BinaryContainsExactOptions::pattern`
```

Comment:
Could you clarify what kinds of objects are used as `pattern` and `value_set` here? Array, Scalar, std::something, etc.?

Also might be worth calling out that these are unary and not binary, as they often appear to be elsewhere. It certainly confused me when I tried to call `isin` like `isin(x, y)` and got the error message that "isin" only takes 1 argument--which doesn't help you understand how to properly construct the call, it just makes you question your sanity.

Reply:
You get the detailed API docs by clicking on the `BinaryContainsExactOptions` hyperlink (in the rendered docs only, though :-)).